mlflow/models/python_api.py

```diff
@@ -152,19 +148,40 @@ def _predict(_input_path: str):
         _predict(input_path)
 
 
+def _get_pyfunc_supported_input_types():
+    # Importing here as the util module depends on optional packages not available in mlflow-skinny
+    from mlflow.models.utils import PyFuncInput
+
+    supported_input_types = []
+    for input_type in get_args(PyFuncInput):
+        if isinstance(input_type, type):
+            supported_input_types.append(input_type)
+        elif isinstance(input_type, ForwardRef):
+            name = input_type.__forward_arg__
+            base_module = importlib.import_module("mlflow.models.utils")
```

Comment:
can we an import statement to import `mlflow.models.utils` instead of `import_module`?

Reply:
Like `import mlflow.models.utils as base_module`, right? What is the difference between two?

Reply:
Exactly. No big difference,  but `import mlflow.models.utils` is more straightforward, doesn't require `importlib`, and easier to be statically analyzed. I use `import_module ` when the module to import is dynamically determined during runtime.

Reply:
Makes sense, thanks for the explanation!